sources/api/netdog/src/main.rs

```diff
@@ -21,10 +21,15 @@ name of the interface, and valid options are "dhcp4" and "dhcp6".  A "?" may be
 to signify that the lease for the protocol is optional and the system shouldn't wait for it.  A
 valid example: `netdog.default-interface=eno1:dhcp4,dhcp6?`.
 
+The subcommand `check-net-config` follows the same logic as `generate-net-config` to find the
+network interface configuration for the host. Rather than generate the configuration, it prints very
+basic information about the found configuration or the error found while attempting to parse the
+file. This command is intended to be run by a user to test network configuration files.
+
 The subcommand `write-resolv-conf` writes the resolv.conf, favoring DNS API settings and
 supplementing any missing settings with DNS settings from the primary interface's DHCP lease.  It
 is meant to be used as a restart command for DNS API settings.
-*/
+ */
```

Comment:
nit: extra space here

Reply:
For some reason my IDE keeps putting this back if I don't watch it. Very strange. I'll fix nonetheless.